alerting/receivers.go

```diff
@@ -186,9 +213,13 @@ func (am *GrafanaAlertmanager) TestReceivers(ctx context.Context, c apimodels.Te
 			return nil
 		})
 	}
-	g.Wait() // nolint
+	err = g.Wait() // nolint
```

Comment:
The linter was complaining about this - I don't think the group can fail (because we handle failures) but worth checking to appease the linter.